src/python_testing/TC_FAN_3_1.py

```diff
@@ -36,16 +36,43 @@
 
 import logging
 import time
+import random
 
 import chip.clusters as Clusters
 from chip.interaction_model import Status
-from chip.testing.matter_testing import MatterBaseTest, async_test_body, default_matter_test_main
+from chip.testing.matter_testing import MatterBaseTest, TestStep, async_test_body, default_matter_test_main
 from mobly import asserts
 
+# import pdb
+
 logger = logging.getLogger(__name__)
 
 
 class TC_FAN_3_1(MatterBaseTest):
+    
+    def steps_TC_FAN_3_1(self):
+        return [TestStep("1", "Commissioning already done."),
+                TestStep("2", "Action",
+                         "Verification"),
```

Comment:
These tests steps should have some better description' Action/Verification is not very useful.